### ISSUE TITLE
subproject?

### ISSUE THREAD

**Issue by alanlenzi** [oracc/editing#8]
I set up a subproject (caspo/akkpm) and adjusted its config.xml file to draw catalog data directly from CDLI (thanks to Eleanor's help). I ran the oracc config in the caspo/akkpm directory. And then I put one bare-bones atf file in the subporject's 00atf folder. I ran the oracc check, build, serve commands in the subproject's directory, but nothing is showing up in the pager. In fact, there is no pager for the subproject. What am I doing wrong?

**Comment by stinney** [comment]
Can you check the #project line in your ATF file and make sure it says #project: caspo/akkpm ?  The symptom you describe is often associated with moving files from project to project without updating the #project line; same for .glo files.

**Comment by alanlenzi** [comment]
Yes, the project line was wrong. But that didn't fix my problem. Still no corpus. Then I realized I had not used the CDLI designation, so I changed that.
BEFORE:
&P269974 = CBS 19842
#project: caspo
AFTER:
&P269974 = PBS 01/1, 02
#project: caspo/akkpm
Still no corpus, after doing oracc check, build, serve commands in the caspo/akkpm directory. I also tried a "build clean." No luck.

**Comment by alanlenzi** [comment]
Nothing shows up at http://oracc.museum.upenn.edu/caspo/akkpm/corpus even though I have applied the fixes above. Any help?

**Comment by alanlenzi** [comment]
My subproject still does not show up. Any ideas what to do?

**Comment by EleanorRobson** [comment]
I can see your subproject at [http://build-oracc.museum.upenn.edu/caspo/akkpm/corpus](http://build-oracc.museum.upenn.edu/caspo/akkpm/corpus) with just one text (PBS 1/1, 02), which I presume corresponds to cbs_19842.atf in your akkpm/00atf folder. Can you be more specific about the problem you're experiencing please?

**Comment by alanlenzi** [comment]
I'm surprised to see it on the build server since I performed the oracc build AND oracc serve commands within the subdirectory of the subproject. I'd like to have it available at http://oracc.museum.upenn.edu/caspo/akkpm/corpus. I can't see it there yet. I'm not sure why. The config.xml file indicates {public}yes{/public} (pointed brackets used appropriately, replaced here for viewing). Does one still have to ask one of the steering committee members to rebuild the project list (per http://oracc.blogspot.com/2017/10/the-new-oracc-serve-command.html)? I am making my list on an ad hoc basis. That's why there is only one text so far.
